src/localization.ts

```diff
@@ -230,6 +230,26 @@ const LANG_DA: Localization = {
   text_fetch_error: "Det lykkedes ikke at oprette forbindelse til",
 };
 
+// Russian
+const LANG_RU: Localization = {
+  text_init: "Инициализация..",
+
+  text_ready: "Антибот проверка",
+  button_start: "Нажмитке чтобы начать проверку",
+
+  text_fetching: "Нажмите для проверки",
+
+  text_solving: "Подтверждение того что вы человек",
+  text_completed: "Я человек",
+
+  text_expired: "Спрок проверки антибот истёк",
+  button_restart: "Начать заново",
+
+  text_error: "Ошибка проверки",
+  button_retry: "Повторить ещё раз",
+  text_fetch_error: "Ошибка подключения",
```

Comment:
Here in English, it says "Failed to connect to" but translation is just "Failed connection" without specifying to. I don't know if it's intended :)